src/towncrier/create.py

```diff
@@ -32,30 +35,38 @@
 )
 @click.option(
     "--edit/--no-edit",
-    default=False,
+    default=None,
     help="Open an editor for writing the newsfragment content.",
-)  # TODO: default should be true
+)
 @click.option(
     "-c",
     "--content",
     type=str,
-    default="Add your info here",
+    default=DEFAULT_CONTENT,
     help="Sets the content of the new fragment.",
 )
-@click.argument("filename")
+@click.option(
+    "--eof-newline/--no-eof-newline",
+    default=None,
+    help="Ensure the content ends with an empty line.",
+)
+@click.argument("filename", default="")
 def _main(
     ctx: click.Context,
     directory: str | None,
     config: str | None,
     filename: str,
-    edit: bool,
+    edit: bool | None,
```

Comment:
Why not leave `edit` as always being boolean ?

It looks like we are using `None` here as a placeholder for the default value.

But maybe it's better to have the explicit default value defined in this way.

```suggestion
    edit: bool = True,
```

Reply:
This is only set to `True` only if the content is still the default content. If a user specifies their own content then, we don't want to default to triggering editing mode. But by leaving it `None`, it makes it possible to provide custom content **and** still explicitly choose to enable editing mode with this flag.

Reply:
We can have it like this... it looks like we are hijacking this variable :)

We should have a separate flag to signal the other use case.

Reply:
It's the same behaviour as any of the other cli settings that also have a configuration setting. If None, fall back to the configuration, otherwise use the value as provided.